### ISSUE TITLE
use boxAnnotation

### ISSUE THREAD

**Issue by Demonhuang** [ABTSoftware/SciChart.Android.Examples#19]
hello ,I find a problem,use boxAnnotation,Anyway, change the drawing order ，boxAnnotation all on the top level，Is there any way to make it draw at the lower level, that is, first

**Comment by andyb1979** [comment]
Yes of course, the order of annotations in the SciChartSurface.annotations collection is the drawing order 

simply change the order in the collection and this will draw annotations earlier

**Comment by Demonhuang** [comment]
> Yes of course, the order of annotations in the SciChartSurface.annotations collection is the drawing order
> 
> simply change the order in the collection and this will draw annotations earlier

I mean ，in suface have moutain and Annotation,No matter how I change the order，Moutain is always below the annotation level，but i hope Moutain is above the annotation level

**Comment by andyb1979** [comment]
OK I understand. 

Try setting [annotation.setAnnotationService(AnnotationSurfaceEnum.BelowChart)](https://www.scichart.com/documentation/android/latest/api/charting/com.scichart.charting.visuals.annotations.IAnnotation.html#com_scichart_charting_visuals_annotations_IAnnotation_setAnnotationSurface_com_scichart_charting_visuals_annotations_AnnotationSurfaceEnum_)

This will place annotations below the series on the chart. However, it will also place annotations below gridlines and axis bands. 

Best regards, 
Andrew

**Comment by Demonhuang** [comment]
It has been solved ，Thank you
